build/package/Installer.MSI.targets

```diff
@@ -146,16 +147,20 @@
     <Target Name="GenerateSdkMSBuildExtensionsNupkg"
             DependsOnTargets="Init;Layout;MsiTargetsSetupInputOutputs;GenerateSdkBundle"
             Condition=" '$(OS)' == 'Windows_NT'"
-            Inputs="$(SdkResolverOutputDirectory)/**/*;
+            Inputs="$(MSBuildExtensionsLayoutDirectory)/**/*;
```

Comment:
The resolver goes in the same nupkg, but now in a subfolder.  From OutputDirectories.props:

```xml
<SdkResolverOutputDirectory>$(MSBuildExtensionsLayoutDirectory)/MSBuildSdkResolver</SdkResolverOutputDirectory>
```